presto-orc/src/main/java/com/facebook/presto/orc/StreamDescriptor.java

```diff
@@ -65,7 +66,12 @@ public int getSequence()
 
     public OrcTypeKind getStreamType()
```

Comment:
getStreamType name doesn't work well with getOrcType. Would you rename getStreamType to getOrcTypeKind in a separate commit?